utilities/src/main/java/io/syndesis/qe/utils/BoxUtils.java

```diff
@@ -0,0 +1,184 @@
+import org.jsoup.select.Elements;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Component;
+
+import com.box.sdk.BoxAPIConnection;
+import com.box.sdk.BoxFile;
+import com.box.sdk.BoxFolder;
+import com.box.sdk.BoxItem;
+
+import javax.annotation.PostConstruct;
+
+import java.security.SecureRandom;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+
+import io.syndesis.qe.accounts.Account;
+import io.syndesis.qe.accounts.AccountsDirectory;
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * Class with various box helper methods.
+ */
+@Slf4j
+@Component
+@Lazy
+public class BoxUtils {
+    private static final String SYNDESIS_BOX_DIRECTORY_ID = "72743300980";
+    private static final String OAUTH_URL = "https://account.box.com/api/oauth2/authorize?response_type=code&client_id=";
+
+    private Account boxAccount;
+    @Getter
+    private BoxAPIConnection boxConnection;
+    @Getter
+    private static String fileId;
+
+    @PostConstruct
+    public void initClient() {
+        log.info("Getting access token");
+        boxAccount = AccountsDirectory.getInstance().getAccount("Box").orElseThrow(() -> new RuntimeException("Unable to find box account"));
+        String accessToken;
+        try {
+            accessToken = getAccessToken();
+        } catch (Exception e) {
+            throw new RuntimeException("Unable to obtain box access token!", e);
+        }
+        boxConnection = new BoxAPIConnection(boxAccount.getProperty("clientId"), boxAccount.getProperty("clientSecret"), accessToken);
+    }
+
+    public void clearBox() {
+        log.info("Clearing Box folder");
+        for (BoxItem.Info itemInfo : new BoxFolder(this.boxConnection, BoxUtils.SYNDESIS_BOX_DIRECTORY_ID)) {
+            log.debug("Deleting file {} (id {})", itemInfo.getName(), itemInfo.getID());
+            new BoxFile(this.boxConnection, itemInfo.getID()).delete();
+        }
+    }
+
+    public void uploadFile(String name, String content) {
+        try {
+            BoxFile.Info info = new BoxFolder(this.boxConnection, SYNDESIS_BOX_DIRECTORY_ID).uploadFile(
+                    IOUtils.toInputStream(content, "UTF-8"), name);
+            fileId = info.getID();
+        } catch (Exception ex) {
+            fail("Unable to upload file to Box: ", ex);
+        }
+    }
+
+    public int getFileCount() {
+        return ((Collection<?>)new BoxFolder(this.boxConnection, SYNDESIS_BOX_DIRECTORY_ID)).size();
+    }
+
+    public BoxFile getFile(String name) {
+        for (BoxItem.Info info : new BoxFolder(this.getBoxConnection(), SYNDESIS_BOX_DIRECTORY_ID)) {
+            if (name.equals(info.getName())) {
+                return new BoxFile(this.getBoxConnection(), info.getID());
+            }
+        }
+        return null;
+    }
+
+    /**
+     * From org.apache.camel.component.box.internal.BoxConnectionHelper
+     * @return box access token
```

Comment:
there is usually one more empty line in between comment and stuff like `@return` arguments, at least my IDE does that